src/deprecates.jl

```diff
@@ -1,6 +1,14 @@
 #### Deprecate on 0.6 (to be removed on 0.7)
 
 @Base.deprecate expected_logdet meanlogdet
+if isdefined(SpecialFunctions, :logabsgamma)
```

Comment:
```suggestion
if !isdefined(SpecialFunctions, :logabsgamma)
```
(and same below)

Reply:
Oh !!!!
Very sorry ! Yeah correcting it.